### PR TITLE
Iterate on landing copy

### PR DESCRIPTION
**Pull request by kytwb**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -205,7 +205,7 @@ <h3>Easily import and create accounts</h3>
                                 <li class="wow fadeInUp" data-wow-duration=".7s" data-wow-delay=".5s"> Import your
                                     accounts from andOTP. </li>
```
Comment (kytwb) [inline index.html:206]: @vantezzen this one needs an iteration as well.

File: index.html [hunk 2]
```diff
@@ -75,10 +75,10 @@
                     <div class="col-xl-7 col-md-6">
                         <div class="slider_text ">
                             <h3 class="wow fadeInDown" data-wow-duration="1s" data-wow-delay=".1s">
-                                All Your TOTP Codes <br />in the menubar
+                                Authentication App<br />For Your Desktop
```
Comment (vantezzen) [inline index.html:78]: I'm not really a fan of this one - I think it doesn't really show that Sauropod is a TOTP app as "Authentification App" is such a broad name

File: index.html [hunk 3]
```diff
@@ -135,13 +123,25 @@ <h3>Easy to use</h3>
                             <img src="img/icon/3.svg" alt="">
                         </div>
                         <h3>
-                            Compatible with all your services
+                            Compatible with all
```
Comment (vantezzen) [inline index.html:126]: Maybe "all your accounts". I feel like there is something missing after the "all"

File: index.html [hunk 4]
```diff
@@ -135,13 +123,25 @@ <h3>Easy to use</h3>
...
+                        <div class="thumb">
+                            <img src="img/icon/2.svg" alt="">
+                        </div>
+                        <h3>Open-Source</h3>
+                        <p>
+                            You can find Sauropod's complete source code on <a
+                                href="https://github.com/vantezzen/sauropod">GitHub</a>.
+                        </p>
+                    </div>
+                </div>
```
Comment (vantezzen) [inline index.html:144]: You're right, this fits much better here

File: index.html [hunk 5]
```diff
@@ -199,13 +199,13 @@ <h3>Enjoy the app</h3>
...
                     <div class="col-xl-5 col-lg-5 col-md-6">
                         <div class="features_info">
-                            <h3>Easily import and create accounts</h3>
+                            <h3>Easily import accounts</h3>
                             <p></p>
                             <ul>
-                                <li class="wow fadeInUp" data-wow-duration=".7s" data-wow-delay=".5s"> Import your
-                                    accounts from andOTP. </li>
+                                <li class="wow fadeInUp" data-wow-duration=".7s" data-wow-delay=".5s"> Import 
+                                    from your existing authentication app. </li>
```
Comment (vantezzen) [inline index.html:206]:
```suggestion
                                    your accounts from your existing authentication app. </li>
```

File: index.html [hunk 6]
```diff
@@ -199,13 +199,13 @@ <h3>Enjoy the app</h3>
...
                                 <li class="wow fadeInUp" data-wow-duration=".8s" data-wow-delay=".6s"> Scan your display
-                                    to find OTP QR Codes. </li>
+                                    to find import QR codes. </li>
```
Comment (vantezzen) [inline index.html:208]:
```suggestion
                                    to import your account from a QR code. </li>
```

File: index.html (hunk 3 above)
Comment (kytwb) [reply index.html:126]:
```suggestion
                            Compatible with all your accounts
```

File: index.html (hunk 2 above)
Comment (kytwb) [reply index.html:78]:
```suggestion
                                All Your TOTP Codes<br />On Your Desktop
```

**Pull request merged**